src/mars.d

```diff
@@ -136,6 +136,7 @@ Where:
   -Hf=<filename>   write 'header' file to filename
   --help           print help and exit
   -I=<directory>   look for imports also in directory
```

Comment:
Actually this documentation is a bit misleading. Both `-Ifoo` and `-I=foo` works, and people will intuitively use `-Ifoo` because it matches what other compilers do.